OpenRA.Mods.Common/Traits/World/Locomotor.cs

```diff
@@ -34,7 +34,9 @@ public enum CellBlocking : byte
 		Empty = 0,
 		HasActor = 1,
 		FreeSubCell = 2,
-		Crushable = 4
+		IsMoving = 4,
+		Crushable = 8,
+		IsTemporaryBlocker = 16
```

Comment:
How about `HasTemporaryBlocker`? `Is` implies that there is only one thing in the cell, which may not be true.